ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -2226,6 +2226,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-openstack,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Do we want to run these tests on all PR? I would say, let's start with false and invoke the tests manually where needed. Later on when we have good ci signal, we can add something like "run_if_changed: .*/on-prem/.*|.*openstack.*" to run on related changes?

Reply:
100%

I'll fix it, thanks